packages/datadog_flutter_plugin/android/src/test/kotlin/com/datadoghq/flutter/DatadogRumPluginTest.kt

```diff
@@ -454,6 +455,33 @@ class DatadogRumPluginTest {
         verify { mockResult.success(null) }
     }
 
+    @Test
+    fun `M call internal updatePerformanceMetrics W updatePerformanceMetrics is called`(
```

Comment:
we need to add the test where one metric is there, but another is missing, because the check is `if (buildTimes != null && rasterTimes != null)`

Reply:
The contract tests do that. They test that the call succeeds if both parameters are present, and fails if either one is missing (it checks once for each parameter).  See: https://github.com/DataDog/dd-sdk-flutter/blob/6d789792ffe212ba8f60c609aa5a96a9a3409afd/packages/datadog_flutter_plugin/android/src/test/kotlin/com/datadoghq/flutter/ContractHelpers.kt#L66